src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/appsettings.Development.json

```diff
@@ -34,5 +34,9 @@
         ]
       }
     }
+  },
+  "fileScanner": {
+    "Url": "localhost",
+    "Port": 3310
```

Comment:
I think this should be a configurable secret? We want to reach out to our openshift clamAV instance.

Reply:
fair enough, moved configuration to secrets